src/fircomb.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{CombFilter, Error, FilterParam, FilterType};
+
```

Comment:
I would recommend trying to condense all of the comb filters into one file only as an option to reduce redundancy.